docs/source/tutorials/cmip6-ecs.ipynb

```diff
@@ -0,0 +1,3690 @@
dnIsA7a7pHUVXO1Kto9kvwqgsanRvjXhIRyzAkAKZNNxKxlX6dUm9nVNBd+P39829vXN2Nf/C05jcc61l/Tr2M2mzrlmgdeYLelK2S9gC0ndJJ0sK0+8WNLDdVwTAGSymu7zarOPZL8KoLGpzX6PY1YASK2sOW4l4yraXOxrvN71T7IMrJMkDXHOfSQrJTxG0lpZ2nVzSaXxF/A87zNJnwVec4OkF5xz30gaK+k059w/Pc8bW8s1AUA2q+0+ryaPZ78KoLEpt9/jmBUA0ibyx61kXKVXPFrZpoLvtw4+zvO8xZJ2lnSX7ErUZbKg1ZuSDpTUTNIaz/M2VfXGnuf9LOnt2M19arsmAMhwNd3nVfX4ZFep2K8CaGxStt/jmBUAai1rjlsJXKXX1NjXiupDB8a+bqkv9Txvmed5v/Q8r5/neU08z+vsed4Fsr5XTtL3NXj/ZbGvLQL31XhNAJDBKtznOefyZPvWEsUGYHiet17SAkktnXNdk7xesn0k+1UAjU2N9q3VwDErANRQNh23ErhKr09iXw92zoX+XzjnWknaU1KRpG+q8VoXxb4+VYP33zX2NXjQMFPSPEmDnHN9kzznsNjXj2vwPgAQVfF92aFJvrePrPz6K8/zNlbzOcn2kexXATQ2tdm3VoZjVgConaw4biVwlUae582U9L6kPpIuT/j2X2RXlR6PRUrlnMtxzrVMfB3n3IWSTpM0RgmBK+fc3kke75xzN0jaXTapcMtoTM/zPEn3xW7+KxhQc84dI2lv2UTDYA8CAMhUL8r2g6c653aK3+mcK5D099jN/yY8J76P/L1zbqvAc/rI9uUbJT0Sv5/9KoBGqMb7Vo5ZAaBeZMVxq7N1IV2cc/0lfSWpk6TXZCOBd5W0nyz9bg/P81bEHttS0hJJH0iaEXuJvSXtIouMHuh53pyE1/dir/O9LE2wjSyTa5is6eVxnue9n/CcprII6h6SfpD0kaResqbwmyTt73net6n6bwAAqeScO1bSsbGbXSQdIrtKPyp233LP865NePyLkoolPStppaSjZZOtXpR0spfwYemcu002zXV+7DFNJJ0iqb2kKz3Puzvh8exXAWS0+t63cswKoDGqxb71t5IGx24Ol7S9LJ4wPXbfF57nPZjwHhl/3ErgKgKccz0l/VWWvtde0iJJr0r6i+d5KwOPy5dFP/eS1CN290zZL9+/Pc9bl+S1b5EFtgZKaiepTJb692HsOUl7Czjnmkn6raTTZb+khZI+lfRnz/Mm1ekHBoB65Jz7P0l/ruQhcz3P65PwnD0l/V52Vb9AdnHgYUl3eZ5XWu4V7DnnSLpC0jayfeuPkm7xPO/NCh7PfhVAxqrvfSvHrAAao5ruW51zn0rat5LHP+Z53rlJ3iejj1sJXAEAAAAAACCS6HEFAAAAAACASCJwBQAAAAAAgEgicAUAAAAAAIBIInAFAAAAAACASCJwBQAAAAAAgEgicAUAAAAAAIBIInAFAAAAAACASCJwBQAAAAAAgEgicAUAAAAAAIBIInAFAAAAAACASCJwBQAAAAAAgEgicAUAAAAAAIBIInAFAADQQJxzLzvnPOfc1ZU8Zmfn3Gbn3GznXOuGXB8AAEDUOM/z0r0GAACARsE5117SeEntJO3sed74hO83l/STpAGS9vU874uGXyUAAEB0kHEFAADQQDzPWyHpPElNJD3lnGua8JDbJA2SdDNBKwAAAAJXAAAADcrzvPck3S1pW0k3xe93zh0u6RJJoyX9n3Muzzl3mXPuG+dcoXNug3PuJ+fcFc65csdwzrlznXMvOedmOeeKYs/50jl3ZrJ1OOc+jZUtNnHO/ck5N9U5t9E592i9/OAAAAC1QKkgAABAA3POFcgCVEMkHSRprKQJklpJGiFplqQ3JB0iaaqkTyUVS9pP0naSnvQ876yE1yySNElWirhIUntJh0vqLunvnuf9MeHxn0raV9KbknaW9I6kpZKWep53W4p/ZAAAgFrJS/cCAAAAGhvP84qdc2dI+lbSY5LGSeos6TLP86Y65/5PFrS6W9KvPM8rlSTnXK6k+yWd75x70fO81wIvO8zzvJnB93HONZEFpH7rnLvP87wFSZbTO/bc5an9KQEAAOqOUkEAAIA08DxvjKQ/yjKiDpP0lud5/42VAV4habGkq+NBq9hzSiVdI8mTdEbC64WCVrH7Nkm6R3ax8oAKlvJHglYAACCqyLgCAABIn1slXS2pi6TrYvcNkpX5TZf0B+dcsucVycoMt3DO9ZL0/tc/UwAAAgZJREFUG1mAqpekZgnP6V7BGr6rzcIBAAAaAoErAACANPE8r8w5tzF2syj2tX3s60BJf67k6S3jG865frIA1FaSRkl6X9IaSaWS+kg6R1LiBMO4xbVZOwAAQEMgcAUAABAta2JfX/E87/hqPufXsoDXeZ7nPRr8hnPuNFngKimPST0AACDC6HEFAAAQLVMkrZa0m3Muv5rPGRD7+lKS7+2bikUBAACkA4ErAACACPE8r0TSfyR1lXSXcy6xV5Wcc12dc9sE7poT+zoy4XGHSLqwflYKAABQ/ygVBAAAiJ6/Sdpe0iWSjnLOfSxpgaROst5Xe0r6vaRJscffK+k8SS84516KPXaYpEMlPS/plAZdPQAAQIoQuAIAAIgYz/M2O+eOlXSmpHMlHSlrxr5M0mxJf5T0VODx45xz+0n6u6TDZcd4YyUdLys7JHAFAAAykqMfJwAAAAAAAKKIHlcAAAAAAACIJAJXAAAAAAAAiCQCVwAAAAAAAIgkAlcAAAAAAACIJAJXAAAAAAAAiCQCVwAAAAAAAIgkAlcAAAAAAACIJAJXAAAAAAAAiCQCVwAAAAAAAIgkAlcAAAAAAACIJAJXAAAAAAAAiCQCVwAAAAAAAIgkAlcAAAAAAACIJAJXAAAAAAAAiCQCVwAAAAAAAIgkAlcAAAAAAACIJAJXAAAAAAAAiKT/B0iPCBR9++tvAAAAAElFTkSuQmCC\n",
```

Comment:
There are a few small things you could do to make this plot nicer:

1. What does the `height = 2.0` in the title mean?
2. Instead of just labeling the y-axis `tas`, how about using the variable long name and including units?
3. "time" is a reasonable x-axis label, but maybe "year" would give more context to the data? You make this change in the next plot